frontend/src/redux/filterSlice.ts

```diff
@@ -8,11 +8,14 @@ export interface FilterState {
   namespaces: Set<string>;
   /** The search string to filter on. */
   search: string;
+  /** The categories to filter on. */
+  categories: string[];
```

Comment:
Categories is something that belongs only to the CRDs, so I don't think we should make it global like this.